packages/gensx/tests/checkpoint.test.tsx

```diff
@@ -363,6 +374,111 @@ suite("checkpoint", () => {
     });
   });
 
+  test("handles out of order node insertion and tree reconstruction", () => {
```

Comment:
can this be a test `suite`, with a few separate tests in it?

Reply:
Done via https://github.com/cortexclick/gensx/pull/170/commits/21a49cf14acc19a824ce38f3c587a63688f5b122